package.json

```
@@ -38,9 +38,14 @@
         "prepare": "rm -rf ./dist && npm run build",
         "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
         "format": "npx prettier . --write",
-        "prepublishOnly": "npm run test"
+        "prepublishOnly": "npm run test",
+        "prepublish": "npm run lint && npm run prepare && npm run test:coverage"
```

```suggestion
        "prepublish": "npm run build npm run lint && npm run prepare && npm run test:coverage"
```

I guess we need this as well 🤔

---

`npm run prepare` already runs `npm run build`, from:
```
"prepare": "rm -rf ./dist && npm run build",
```